Anchor.toml

```diff
@@ -1,23 +1,26 @@
 [toolchain]
 
 [features]
-seeds = false
-resultion = true
+resolution = true
 skip-lint = false
 
+[programs.localnet]
+wen_new_standard = "wns1gDLt8fgLcGhWi5MqAqgXpwEP1JftKE9eZnXS1HM"
+wen_royalty_distribution = "diste3nXmK7ddDTs1zb6uday6j4etCa9RChD8fJ1xay"
+wen_transfer_guard = "4MrF8SuFtwGg53bL6KziCe7suvKqUoBg7o7Lnix1Ton5"
```

Comment:
@kevinrodriguez-io is this a key you generate? Please get a initial deploy onto devnet once we finish up CR. We'll redeploy under a new address before launching to mainnet.